src/plugins/custom/workflow.ts

```diff
@@ -0,0 +1,33 @@
+import fp from 'fastify-plugin';
+import { MultiOriginTransition, Transition, Workflow } from '@jean-michelet/workflow';
```

Comment:
This is a component I created inspired by [symfony/workflow](https://symfony.com/doc/current/components/workflow.html). You can read the documentation here, I am willing to create an official Fastify plugin if you are interested.

Reply:
I still thing, this shouldn't be part of the Demo repo.